src/js/ad-servers/gpt.js

```diff
@@ -238,6 +239,13 @@ function onRenderEnded(event) {
 	const iframeId = `google_ads_iframe_${gptSlotId.getId()}`;
 	data.type = domId.pop();
 	data.name = domId.join('-');
+	data.size = event.size && event.size.length ? event.size.toString() : '';
```

Comment:
```suggestion
	data.size = event.size && event.size.length ? event.size.join() : '';
```

Reply:
Calling join() on an empty - zero lenght - array returns an empty string, so we can aboid to check for the array length
```suggestion
	data.size = event.size && event.size.join()';
```